### ISSUE TITLE
[IntelliJ-ThirdParty][A11yMAS][Accessibility][Keyboard][MAS2.1.1][IntelliJ][Wipro]: "Show"combo box and "Sort by" button is not accessible with keyboard in Plugins dialog.

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1430]
**Repro Steps:**

1. Open and Start IntelliJ IDEA.
2. IntelliJ IDEA start screen appears, Navigate and select  Plugins under Configure combo box.
3. Plugins dialog will open.
4. In Plugins dialog navigate to "Show" combo box" and "sort by" button and try to access with keyboard.

**Expected Result:**
User should be able to access the "Show" combo box and "Sort by" button using keyboard.

**Actual Result:**
"Show" combo box and "Sort by" button is not accessible with keyboard.

**Platform:**
IntelliJ IDEA 2017.3.2 (Community edition)
Build# IC-173.4127.27
JRE:1.8.0_152-release-1024-b8amd64
JVM: OpenJDK 64 Bit Server VM by Jetbrains.
**OSBuild Version:** Version 1709(Os Build 16299.192)

![show combobox and sort by button is not accessible with keyboard](https://user-images.githubusercontent.com/22956130/35725395-1ed64b9a-07b6-11e8-86a9-39d79c5650dc.png)

**Comment by Eskibear** [comment]
It's the IDE's own dialog, it's out of the toolkit's scope.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749859
